common/headers/version_checker.go

```diff
@@ -139,7 +139,6 @@ func SetVersionsForTests(ctx context.Context, clientVersion, clientName, support
 
 // ClientSupported returns an error if client is unsupported, nil otherwise.
 func (vc *versionChecker) ClientSupported(ctx context.Context) error {
-
```

Comment:
I guess it is some autoformaing on your side. Can you revert all new lines removal from this change?